src/registrar/templates/application_form.html

```diff
@@ -70,12 +70,18 @@ <h1> {{form_titles|get_item:steps.current}} </h1>
               class="usa-button"
             >Submit your domain request</button>
             {% endif %}
+            {% comment %}
```

Comment:
**Suggestion**: In order to keep our templates simple, I would like to see this code removed rather than commented out. If we want to use this again in the future, we have Git history to get it back.

Reply:
Done. 👍